sphinx/source/quickstart/build.rst

```diff
@@ -44,11 +44,9 @@ The full list of build switches can be obtained by running:
 * **BUILD_TESTS**: Boolean. Build all tests for CCF. Default to ON.
 * **BUILD_SMALLBANK**: Boolean. Build SmallBank performance benchmark. Default to OFF.
 * **CLIENT_MBEDTLS_PREFIX**: Path. Prefix to mbedtls install to be used by test clients. Default to ``/usr/local``.
-* **SERVICE_IDENTITY_CURVE_CHOICE**: String, one of ``secp384r1``, ``secp256k1_mbedtls``, ``secp256k1_bitcoin``. Elliptic curve to use for CCF network and node identities. Defaults to ``secp384r1``.
```

Comment:
Worth updating https://microsoft.github.io/CCF/developers/cryptography.html to mention this?

Reply:
I don't see any reference to this option there, not sure what should be changed in the docs?

Reply:
I guess that bit:

```
They can be configured to use one of the following implementations:
```

Reply:
I meant mentioning network and node identities in our cryptography page (which is rather generic atm).

Reply:
Have added a more up-to-date comment on service-curves vs user-curves, and how service-curve is selected.